src/clarity-angular/popover/signpost/signpost.specs.ts

```diff
@@ -11,385 +11,503 @@ import { Signpost } from "./signpost";
+                let signpostToggle: HTMLElement = clarityElement.querySelector(".btn");
+                let signpostContent: HTMLElement;
+
+                // Test we have a trigger
+                expect(signpostToggle).not.toBeNull();
+
+                signpostToggle.click();
+                // Test that it doesn't show until changes detected
+                signpostContent = clarityElement.querySelector(".signpost-content");
+                expect(signpostContent).toBeNull();
+
+                fixture.detectChanges();
+                // Test it shows after changes
+                signpostContent = clarityElement.querySelector(".signpost-content");
+                expect(signpostContent).not.toBeNull();
+                expect(signpost.ifOpenService.open).toBe(true);
+
+                // Hide it
+                signpostToggle.click();
+                fixture.detectChanges();
+                signpostContent = clarityElement.querySelector(".signpost-content");
+                // Test that its hidden
+                expect(signpostContent).toBeNull();
+                expect(signpost.ifOpenService.open).toBe(false);
+            });
 
-        it("can hide/show content", function() {
-            let signpostToggle: HTMLElement = clarityElement.querySelector(".btn");
-            let signpostContent: HTMLElement;
-
-            // Test we have a trigger
-            expect(signpostToggle).not.toBeNull();
-
-            signpostToggle.click();
-            // Test that it doesn't show until changes detected
-            signpostContent = clarityElement.querySelector(".signpost-content");
-            expect(signpostContent).toBeNull();
-
-            fixture.detectChanges();
-            // Test it shows after changes
-            signpostContent = clarityElement.querySelector(".signpost-content");
-            expect(signpostContent).not.toBeNull();
-            expect(signpost.ifOpenService.open).toBe(true);
-
-            // Hide it
-            signpostToggle.click();
-            fixture.detectChanges();
-            signpostContent = clarityElement.querySelector(".signpost-content");
-            // Test that its hidden
-            expect(signpostContent).toBeNull();
-            expect(signpost.ifOpenService.open).toBe(false);
-        });
+            it("takes an input for position", function() {
+                fixture.componentInstance.position = "top-middle";
+                fixture.detectChanges();
+                expect(signpost.signpostPosition).toBe("top-middle");
+            });
 
-        it("takes an input for position", function() {
-            fixture.componentInstance.position = "top-middle";
-            fixture.detectChanges();
-            expect(signpost.signpostPosition).toBe("top-middle");
-        });
+            it("does not allow multiple open poovers", function() {
+                expect(signpost.signpostOptions.allowMultipleOpen).toBe(false);
+            });
```

Comment:
Just re-read the spec and I have a test that makes sure we configure the popover to `            expect(signpost.signpostOptions.allowMultipleOpen).toBe(false);` [here](https://github.com/vmware/clarity/blob/master/src/clarity-angular/popover/signpost/signpost.specs.ts#L76)

Since this is tested in the popover spec I think it covers it. Are you ok with that?